clients/unshaded/src/main/java/tachyon/client/file/options/CreateOptions.java

```diff
@@ -143,6 +143,15 @@ public UnderStorageType getUnderStorageType() {
   }
 
   /**
+   * @return the name : value pairs for all the fields
+   */
+  @Override
+  public String toString() {
+    return "CreateTOptions{mBlockSizeBytes=" + mBlockSizeBytes + ",mRecursive=" + mRecursive
```

Comment:
@miasantreble I think having spaces required to make the output human friendly. How about making another method for the tests?